bigbluebutton-tests/playwright/layouts/layouts.spec.js

```diff
@@ -0,0 +1,20 @@
+const { test } = require('@playwright/test');
```

Comment:
```
const { test } = require('@playwright/test');
const { FocusOnPresentation } = require('./focusOnPresentation');
const { FocusOnVideo } = require('./focusOnVideo');
const { MultiUsers } = require('../user/multiusers');
const { encodeCustomParams } = require('../customparameters/util');

const CUSTOM_MEETING_ID = 'layout_management_meeting';
const CUSTOM_STYLE = `userdata-bbb_custom_style=.presentationUploaderToast{display: none;}.currentPresentationToast{display:none;}`;

// Initialize mod page with provided params
const initModPage = async (page, context, options) => {
  const focus = new FocusOnPresentation(browser, context);
  await focus.initModPage(page, true, { customMeetingId: CUSTOM_MEETING_ID, customParameter: encodeCustomParams(CUSTOM_STYLE) });
  await focus.initModPage2(true, context, { customParameter: encodeCustomParams(CUSTOM_STYLE) });
};

test.describe.parallel('Layout management', () => {
  test('Focus on presentation', async ({ browser, context, page }) => {
    await initModPage(page, context);
    const focusOnPresentation = new FocusOnPresentation(browser, context);
    await focusOnPresentation.test();
  });
  
  test('Focus on video', async ({ browser, context, page }) => {
    const focusOnVideo = new FocusOnVideo(browser, context);
    await initModPage(page, context);
    await focusOnVideo.test();
  });
});
```

I thought a bit differently. What do you think @MaximKhlobystov ?

Reply:
I don't think we should create multiple instances of FocusOnPresentation. But the idea of reusing the constants is good. Thanks for the feedback!